docs/telemetry.md

```diff
@@ -22,32 +92,149 @@ An example of the data collected is the following:
 {"key" : "hardware_arch","value" : "x86_64 x86_64"}]}]
 ```
 
+## Telemetry Agent
+
+The Percona Telemetry agent functions as a dedicated OS daemon process. The service name on the operating system is `percona-telemetry-agent`. The agent can create, read, write, and delete files in the `${telemetry root path}` and only interacts with JSON files.
+
+The agent sends the collected information as a file to the Percona Platform every 24 hours. If the attempt fails, the agent tries again after a defined period with a limit of five attempts. After a successful transmission, the agent deletes the file.
+
+The agent does not send anything if no Percona-specific files are in the target directory.
+
+The following is an example of a telemetry agent payload:
+
+```json
+{
+  "reports": [
+    {
+      "id": "B5BDC47B-B717-4EF5-AEDF-41A17C9C18BB",
+      "createTime": "2023-09-01T10:56:49Z",
+      "instanceId": "B5BDC47B-B717-4EF5-AEDF-41A17C9C18BA",
+      "productFamily": "PRODUCT_FAMILY_PS",
+      "metrics": [
+        {
+          "key": "OS",
+          "value": "Ubuntu"
+        },
+        {
+          "key": "pillar_version",
+          "value": "8.0.33-25-1"
+        }
+      ]
+    }
+  ]
+}
+```
+
+The agent sends information about the database and metrics.
+
+| Key | Description |
+|---|---|
+| "id" | A generated Universally Unique Identifier (UUID) version 4 |
+| "createTime" | UNIX timestamp |
+| "instanceId" | The DB Host ID. The value can be taken from the `instanceId`, the `/usr/local/percona/telemetry_uuid` or generated as a UUID version 4 if the file is absent. |
+| "productFamily" | The value from the file path |
+| "metrics" | An array of key:value pairs collected from the Metrics file.
+
+The following operating system-level metrics are sent with each check:
+
+| Key | Description |
+|---|---|
+| "OS" | The name of the operating system |
+| "hardware_arch" | The type of processed used in the environment |
+| "deployment" | How the application was deployed. <br> The possible values could be "PACKAGE" or "DOCKER". |
+| "installed_packages" | A list of the installed Percona packages.|
```

Comment:
For PG we send not only Percona packages. Not sure this should be updated for PS docs or not,